nazgul/api.py

```diff
@@ -73,7 +73,7 @@ def location_show():
     xml = xmlrenderer.XMLRenderer()
 
     product = request.args.get("product")
-    fuzzy = request.args.get("fuzzy") == "True"
+    fuzzy = request.args.get("fuzzy") in ["True", "true"]
```

Comment:
```suggestion
    fuzzy = request.args.get("fuzzy").lower() == "true"
```

We could do it like that instead, so we catch all the permutations of "True".